templates/home.html

```diff
@@ -16,6 +16,7 @@
               {% endif %}
           {% endfor %}
           <div class="topnav-right">
+            <a href="{{ attack_url }}">ATT&CK</a>
```

Comment:
per my comment about the gui_api, this change would also be removed, as having an "address" in the hook file will immediately give the plugin a navbar entry. i could see adding an option after address, like "config", "plugin" or "mode" -- see me on this.